scripts/install/opt/seagate/cortx/motr/config/be_resize.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+# set -x
+
+current_time=$(date +"%Y_%m_%d__%H_%M_%S")
+LOG_DIR="/var/log/seagate/motr"
+mkdir -p $LOG_DIR
+LOG_FILE="$LOG_DIR/be_log_resize_$current_time.log"
+touch $LOG_FILE
+MOTR_CONF_FILE="/etc/sysconfig/motr"
+M0BETOOL="m0betool"
+MD_DIR="/var/motr"
+BE_REPLY="be_recovery_run"
+BE_RESIZE="be_log_resize"
+
+local_md_volume=""
+remote_md_volume=""
+new_be_log_size=""
+current_be_log_size=""
+local_node=""
+remote_node=""
+loca_ios_fid=""
+remote_ios_fid=""
+
+be_resize_log() {
+    echo "[`date`]: $@" 2>&1 | tee -a ${LOG_FILE}
+}
+
+be_resize_soft_exit() { be_resize_log "$@" ; exit 0; }
+
+be_resize_hard_exit() { be_resize_log "$@" ; exit 1; }
+
+# Verifying cluster status
+check_cluster_status() {
+	if  [[ $((hctl status) 2>&1 | grep "Cluster is not running") ]]; then
+		be_resize_log "cluster is not running"
+	else
+		be_resize_hard_exit "cluster should be stop for BE log resize"
+	fi
+}
+
+# Extract current BE Log file size
+get_be_log_size() {
+	new_be_log_size=`grep -w MOTR_M0D_BELOG_SIZE $MOTR_CONF_FILE | grep -v '#' | cut -d"=" -f2`
+	if [[ -z $new_be_log_size ]]; then
+		be_resize_soft_exit "BELOG Size is not configured"
+	fi
+	be_resize_log "Given BE log size: $new_be_log_size"
+}
+
+# The function will execute command on the local node
+run_cmd_on_local_node() {
+    local cmd=$1
+    be_resize_log "[`date`]: Running '$cmd'" >> ${LOG_FILE}
+    eval "$cmd" 2>&1 | tee -a ${LOG_FILE}
+    return ${PIPESTATUS[0]} # return exit status of previous command
+}
+
+# The function will execute command on the remote node
+run_cmd_on_remote_node() {
+    local cmd=$1
+    be_resize_log "[`date`]: Running 'ssh root@$remote_node $cmd'" >> ${LOG_FILE}
+    ssh root@$remote_node "$cmd" 2>&1 | tee -a ${LOG_FILE}
+    return ${PIPESTATUS[0]} # return exit status of previous command
+}
+
+# Mount metadata devices on local and remote nodes
+mount_md_device() {
+
+	if [[ $(cat /var/lib/hare/node-name) == "srvnode-1" ]]; then
+		local_node="srvnode-1"
+		remote_node="srvnode-2"
+		local_md_volume=$(grep left-volume /opt/seagate/cortx/ha/conf/build-ees-ha-args.yaml |\
+				    awk '{ print $2 }')
+		remote_md_volume=$(grep right-volume /opt/seagate/cortx/ha/conf/build-ees-ha-args.yaml |\
+				    awk '{ print $2 }')
+
+	elif [[ $(cat /var/lib/hare/node-name) == "srvnode-2" ]]; then
+		local_node="srvnode-2"
+		remote_node="srvnode-1"
+		local_md_volume=$(grep right-volume /opt/seagate/cortx/ha/conf/build-ees-ha-args.yaml |\
+				    awk '{ print $2 }')
+		remote_md_volume=$(grep left-volume /opt/seagate/cortx/ha/conf/build-ees-ha-args.yaml |\
+				    awk '{ print $2 }')
+	else
+		be_resize_hard_exit "ERROR: node-name is not avaialble"
```

Comment:
This is a valid case for LDR R2, needs to be handled, add a todo here.

Reply:
Done.